genie-client/src/test/java/com/netflix/genie/client/GenieClientsIntegrationTestsBase.java

```diff
@@ -84,12 +84,17 @@ Cluster constructClusterDTO(final String id) {
         configList.add("config1");
         configList.add("configs2");
 
+        final Set<String> dependenciesList = new HashSet<>();
```

Comment:
Ok, went with what the rest of these tests were doing.
In the first commit ("test improvements") of the latest version of this PR, I went through **all** tests and replaced this style with `Sets.newHashSet`